### DISCUSSION TITLE
macOS: Can brew-installed py find and use brew-installed python 3.10

### DISCUSSION THREAD

**Discussion by cclauss** [brettcannon/python-launcher#161] [Q&A] [answered]
Is there a way to get `py` to find and use Python 3.10 on macOS?

% `brew install python-launcher python@3.10`
% `/opt/homebrew/opt/python@3.10/bin/python3.10 -VV`
```
Python 3.10.0 (default, Oct 12 2021, 22:37:59) [Clang 13.0.0 (clang-1300.0.29.3)]
```
% `py --list`
```
 3.9 │ /opt/homebrew/opt/python@3.9/bin/python3.9
 2.7 │ /usr/bin/python2.7
```

**Comment by brettcannon** [comment, marked as answer]
If you do `brew link --overwrite python@3.10` then it will work. The issue is that by having `python@3.9` installed, Homebrew won't symlink into `/usr/local/bin` because it would overwrite `python3`. By forcing the linking you reset `python3`, but more importantly for the Python Launcher you get `python3.10` into `/usr/local/bin`.

**Reply by cclauss** [reply]
Perfect.  Thanks much.
